packages/protocol-sdk/src/constants.ts

```diff
@@ -0,0 +1,22 @@
+export const ZORA_API_BASE = "https://api.zora.co/";
+export const OPEN_EDITION_MINT_SIZE = BigInt("18446744073709551615");
+
+// Subgraph base settings
+const CONFIG_BASE =
+  "https://api.goldsky.com/api/public/project_clhk16b61ay9t49vm6ntn4mkz/subgraphs";
+
+function getSubgraph(name: string, version: string): string {
+  return `${CONFIG_BASE}/${name}/${version}/gn`;
+}
+
+export const ZORA_SUBGRAPH_URLS: Record<number, string> = {
```

Comment:
do you thin the subgraph stuff can be broken into another PR? the pr is getting kind of big to review

Reply:
It's kinda needed here – I broke out some other things. Since it's new code I think it's okay for now?

Reply:
yeah i realize its fine, sounds good